ocs/ocp.py

```diff
@@ -0,0 +1,156 @@
+"""
+General OCP object
+"""
+import os
+import logging
+import yaml
+
+from munch import munchify
+from utility.utils import TimeoutSampler
+from utility.utils import run_cmd
+
+log = logging.getLogger(__name__)
+
+
+class OCP(object):
+    """
+    A basic OCP object to run basic 'oc' commands
+    """
+
+    def __init__(self, api_version='v1', kind='Service', namespace=None):
+        """
+        Initializer function
+
+        Args:
+            api_version (str): TBD
+            kind (str): TBD
+            namespace (str): The name of the namespace to use
+        """
+        self._api_version = api_version
+        self._kind = kind
+        self._namespace = namespace
+
+    @property
+    def api_version(self):
+        return self._api_version
+
+    @property
+    def kind(self):
+        return self._kind
+
+    @property
+    def namespace(self):
+        return self._namespace
+
+    def exec_oc_cmd(self, command):
+        """
+        Executing 'oc' command
+
+        Args:
+            command (str): The command to execute (e.g. create -f file.yaml)
+                without the initial 'oc' at the beginning
+
+        Returns:
+            Munch Obj: this object represents a returned yaml file
+        """
+        oc_cmd = "oc "
```

Comment:
Just FYI: this should be later modified to use `get_openshift_client()` function from PR https://github.com/red-hat-storage/ocs-ci/pull/49 (but I'm not sure, which PR will be merged earlier).

Reply:
+1.
If you can open an issue for this it will be great so we won't forget to fix it